src/nvim/memline.c

```diff
@@ -1441,17 +1441,17 @@ recover_names (
  * Append the full path to name with path separators made into percent
  * signs, to dir. An unnamed buffer is handled as "" (<currentdir>/"")
  */
-static char_u *make_percent_swname(char_u *dir, char_u *name)
+static char *make_percent_swname(char *dir, char *name)
```

Comment:
Not easily.  I'm working on a separate PR to do that, as it requires a chain of changes in other functions that wouldn't integrate simply into this PR.